backend/app/routes/broadcast/getBroadcasts.js

```diff
@@ -0,0 +1,43 @@
+const to = require('await-to-js').default;
+const Broadcast = require('../../models/Broadcast');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const page = parseInt(req.query.page, 10);
+  const limit = parseInt(req.query.limit, 10);
```

Comment:
use constants here

Reply:
I didn't get your point. please elaborate.

Reply:
instead of hardcoding the values here declare them in constants file

Reply:
here 10 is passed to parseInt function so that the parsed value is a base 10 number and does not need to be changed.